editor/plugins/audio_stream_editor_plugin.cpp

```diff
@@ -105,6 +105,7 @@ void AudioStreamEditor::_audio_changed() {
 
 void AudioStreamEditor::_play() {
 	if (_player->is_playing()) {
+		_pausing = true;
```

Comment:
It might be good to add a comment as to *why* we are setting `_pausing` to `true` here. It doesn't seem obvious to me.